dask/dataframe/io/__init__.py

```diff
@@ -7,6 +7,7 @@
     from_bcolz,
     from_dask_array,
     from_delayed,
+    from_map,
```

Comment:
You'll need to add to the API docs as well.

Reply:
Sounds good. Do you think I should also add a sub-section (within `Creating`) to `dataframe-create.rst` on "Mapping from a function" (or something like that)?

Reply:
I think it can just go in dataframe-api.rst #create-dataframes and yeah a new section in dataframe-create.rst #creating too I guess.